_faqs/01-20-what-tools-do-you-recommend-for-log-and-metrics-collection.md

```diff
@@ -0,0 +1,6 @@
+---
+question: What tools do you recommend for log and metrics collection?
+category: General
+---
+
+OpenSearch is supported by a range of tools like Beats, Fluentd, Fluent Bit, and OpenTelemetry Collector. OpenSearch believes in multiple options, but will focus effort on improving Data Prepper, Fluentd, and FluentBit support going forward. OpenSearch customers who are using Beats <= 7.12.x as an agent tool, and considering open source alternatives, should migrate to Fluent Bit >= 1.9 or Open Telemetry Collector. Beats version >= 7.13 does not support OpenSearch. 
```

Comment:
Don't use the term 'customers' - this is open source software.

I'm also not a fan of OpenSearch believing things :) OpenSearch doesn't have a brain.